src/URDFConverter/UrdfUnity/UrdfUnity/Util/EnumUtils.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace UrdfUnity.Util
+{
+    /// <summary>
+    /// This is a utility class for working with enums 
+    /// </summary>
+    public static class EnumUtils
+    {
+        /// <summary>
+        /// This provides an assesible method for parsing a string into
```

Comment:
nitpick++; 

<summary>
"Parses the enum type value represented by the provided string name."
</summary>
<typeparam name="T">Type of enum being parsed</>
<param name="value">S...</>